apps/rush-lib/src/cli/actions/InitAutoinstaller.ts

```diff
@@ -0,0 +1,63 @@
+import * as colors from 'colors';
```

Comment:
Copyright?

Reply:
Did a big search+replace across the repo. We should probably enable the [eslint-plugin-notice](https://github.com/nickdeis/eslint-plugin-notice) lint rule.